drink.py

```diff
@@ -60,15 +66,42 @@ def order_drink(drink, retry=True):
     if r.status_code == 403 and retry:
         refresh_token()
         return order_drink(drink, retry=False)
+
     if not r.ok:
-        print(str(r.status_code) + ": " + r.text, file=sys.stderr)
-        sys.exit(1)
-    print(r.text)
+        if r.status_code == 400:
```

Comment:
400 is a magic number

Reply:
fixed: now checking for "Unknown beverage" instead of 400